truffle-config.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+  networks: {
+    local: {
+      host: "127.0.0.1",
+      port: 8545,
+      network_id: "*",
+      gas: 0xffffffffffffffffffffff,
+    },
+    coverage: {
+      host: "localhost",
+      network_id: "*",
+      port: 8555,
+      gas: 0xffffffffffffffffffffff,
+      gasPrice: 0x01,
+    },
+  },
+  mocha: {
+    reporter: "eth-gas-reporter",
+    reporterOptions: {
+      currency: "USD",
+      gasPrice: 2,
```

Comment:
```suggestion
      gasPrice: 20,
```
Today the gas price is 35. Because it changes a lot maybe we could used the same amount as in other projects.

Reply:
I actually multiplied all the values by 10, so the analisis gas analysis is still correct. Changin this now